### PR TITLE
Add @FGRibreau :)

### PR DESCRIPTION
**Pull request by FGRibreau**

### CODE REVIEW

**Comment by dadoonet** [comment]
Could you rebase your PR please?

**Review by dadoonet — DISMISSED** [review]

File: baggers.js [hunk 1]
```diff
@@ -16617,6 +16617,63 @@ var data = {
...
     },
+    {
+      since: '2018-01-01',
+      name: 'Francois-Guillaume Ribreau',
+      bio: 'Full Stack CTO, Startup Advisor, Consultant, Hacker, Maker',
+      picture:
+        'https://media.licdn.com/dms/image/C4D03AQFYw_-F2gSwlg/profile-displayphoto-shrink_200_200/0?e=1560988800&v=beta&t=Doqc2flLGqxUlCWNzWEGpALQ7HGa2kBqTbiVu1Cqn2Q',
+      websites: [
+        {
+          name: 'Site',
```
Comment (dadoonet) [inline baggers.js]: s/Site/Web according to https://github.com/brownbaglunch/bblfr_data#websites

File: baggers.js [hunk 2]
```diff
@@ -16617,6 +16617,63 @@ var data = {
...
+          url: 'https://fgribreau.com/',
+        },
+        {
+          name: 'Livre',
```
Comment (dadoonet) [inline baggers.js]: s/Livre/Book according to https://github.com/brownbaglunch/bblfr_data#websites

**Review by dadoonet — CHANGES_REQUESTED** [review]
Could you modify your PR and rebase? Thanks!

File: baggers.js (hunk 1 above)
Comment (linsolas) [inline baggers.js]:
```suggestion
          name: 'Website',
```

File: baggers.js (hunk 2 above)
Comment (linsolas) [inline baggers.js]:
```suggestion
          name: 'Book',
```

File: baggers.js (hunk 1 above)
Comment (FGRibreau) [reply baggers.js]: fixed :+1:

**Comment by FGRibreau** [comment]
@dadoonet fixed & rebased :)

**Review by dadoonet — APPROVED** [review]

**Pull request merged**